### ISSUE TITLE
Make finer-grained spectest targets

### ISSUE THREAD

**Issue by tbrowder** [rakudo/rakudo#2798]
## The Problem

Running a  full spectest on a new branch will show no errors with a modified existing roast standard test, but trying to run the tests in its S* directory or as an individual test fails.

That is exasperating when trying to test during a core change because the existing spectest takes so long.  A finer-grained spectest for use before a PR would be very usefule to me.

Alternatively, publishing the exact ./perl6 command for local testing of rakudo with single tests in t/spec/S* would be just as useful.

## Expected Behavior

I would like to have a make target something like this:

```
make spectest-S26-documentation
```

**Comment by AlexDaniel** [comment]
Pretty sure you can already do something like:
```bash
make t/spec/S26-documentation/05-comment.t
```

**Comment by tbrowder** [comment]
@AlexDaniel, you are right, and it works as expected!

Where is that documented (pardon my faulty memory, but I think this is old ground and I just can't find it referenced)?

**Comment by AlexDaniel** [comment]
Don't think it's documented anywhere, but see here: https://github.com/rakudo/rakudo#building-and-installing-rakudo
Maybe add a sentence or two saying that if you want to run a single test, then you can do that.

**Comment by tbrowder** [comment]
Closed with commit 68adad469c7c3bb4cc9da73149e9eff0f8370ca1.
